example.env

```diff
@@ -0,0 +1,4 @@
+NETBIRD_MGMT_PORT=33071
+AUTH_AUDIENCE="https://***********.auth0.com/api/v2/" 
+AUTH_AUTHORITY=https://***********.auth0.com
+AUTH_CLIENT_ID=*****************************
```

Comment:
Same as with gitpod.
I think that the dev flow improvement should come separately from the UI Changes